### DISCUSSION TITLE
Make 1.4 releases available in conda-forge

### DISCUSSION THREAD

**Discussion by juanfcocontreras** [zauberzeug/nicegui#1999] [Ideas / Feature Requests]
nicegui seems to have stopped updating in conda-forge (1.3.18 is the latest version). Is there a particular reason for this?

**Comment by rodja** [comment]
Yes, we are aware of that. We just have not found the time to update the dependencies. I'll convert it to a feature request. Maybe someone can help out?

**Reply by juanfcocontreras** [reply]
I think this is done, right?

Thank you so much!

**Reply by rodja** [reply]
You are welcome :-)
